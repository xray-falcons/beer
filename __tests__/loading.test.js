import 'react-native';
import React from 'react';
import Loading from "../components/beer-list";

import renderer from 'react-test-renderer';

describe('<Loading />', () => {


    it('renders correctly', () => {

        const tree = renderer.create(
            <Loading />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
    it('renders the ActivityIndicator component', () => {
        const ActivityIndicator = require('ActivityIndicator');
        const tree = renderer
            .create(<ActivityIndicator animating={true} size="large" />)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });

});
