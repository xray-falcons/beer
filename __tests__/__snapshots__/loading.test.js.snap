// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders correctly 1`] = `
<ViewManagerAdapter_ExpoLinearGradient
  proxiedProperties={
    Object {
      "colors": Array [
        4290998025,
        4293835275,
      ],
      "endPoint": undefined,
      "locations": undefined,
      "startPoint": undefined,
    }
  }
  style={
    Object {
      "flex": 1,
      "height": "100%",
      "left": 0,
      "right": 0,
      "top": 0,
      "width": "100%",
    }
  }
>
  <RCTScrollView
    data={Array []}
    disableVirtualization={false}
    getItem={[Function]}
    getItemCount={[Function]}
    horizontal={false}
    initialNumToRender={10}
    keyExtractor={[Function]}
    maxToRenderPerBatch={10}
    numColumns={2}
    onContentSizeChange={[Function]}
    onEndReachedThreshold={2}
    onLayout={[Function]}
    onMomentumScrollEnd={[Function]}
    onScroll={[Function]}
    onScrollBeginDrag={[Function]}
    onScrollEndDrag={[Function]}
    removeClippedSubviews={false}
    renderItem={[Function]}
    scrollEventThrottle={50}
    stickyHeaderIndices={Array []}
    updateCellsBatchingPeriod={50}
    viewabilityConfigCallbackPairs={Array []}
    windowSize={21}
  >
    <View />
  </RCTScrollView>
</ViewManagerAdapter_ExpoLinearGradient>
`;

exports[`<Loading /> renders the ActivityIndicator component 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="large"
/>
`;
